mcp_servers/app/main.py

```
@@ -1314,3 +1314,249 @@ async def text_to_graphql_query(question: str, network: str = "mainnet") -> Dict
+            }
+        else:
+            logger.error(f"❌ GET_TOKEN_PRICE TOOL: Failed to fetch price for {token_id}")
+            return {
+                "success": False,
+                "token_id": token_id,
+                "error": "Failed to fetch token price from SaucerSwap",
+                "network": network
+            }
+
+    except SDKError as e:
+        logger.error(f"❌ GET_TOKEN_PRICE TOOL: SDKError: {str(e)}", exc_info=True)
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)